src/main/java/org/folio/config/KafkaConfiguration.java

```diff
@@ -0,0 +1,36 @@
+package org.folio.config;
+
+import org.folio.kafka.KafkaConfig;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class KafkaConfiguration {
+
+  @Value("${KAFKA_HOST:kafka}")
```

Comment:
forgot to mention earlier, please also update reame for both invoice-storage and orders-storage that these modules now work with kafka and appropriate env variables, you can use example from some other module that already uses kafka